spec.md

```diff
@@ -2,6 +2,15 @@
 
 The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
 
+# Abstract
+
+The OCI Runtime Specification aims to specify the inputs and outputs for running containers on various platforms.
+
+A container's input or configuration is specified as the `config.json` for the supported platforms and details fields that enable containers to run on that platform.
+It contains host specific information for the creation of a container.
```

Comment:
I think you want “host specific” → “host-specific”, although `master` has inconsistent hyphenation for this.